tools/code_coverage.php

```diff
@@ -0,0 +1,154 @@
+<?php
+/*
+ * This tool is designed to generate code coverage reports for the unit testing suite of LORIS.
+ *
+ * Please run this command from the tools directory -- var/www/loris/tools
+ *
+ * First, this tool will run all the unit tests on the particular branch. This will automatically
+ * generate HTML reports (configured in the phpunit.xml file). Then, the generated files will be
+ * moved to the directory "htdocs/log/codeCoverage/<date>", where the date value is the current date. Finally,
+ * all the HTML reports are edited to include the date they were generated at the top of the page.
+ *
+ * To view the generated HTML reports, go to https://<your-loris-url>/log/codeCoverage/<date>/index.html.
+ * This will take you to the main coverage page. From here, you may navigate to the dashboard or
+ * to the pages for the individual classes under the php/libraries directory.
+ *
+*/
+
+/*
+ * Step 1: Running unit test suite and generating HTML reports
+ *
+ * The unit test suite is run with the --coverage-html tag,
+ * which is configured to generate the HTML reports
+ */
+error_reporting(E_ERROR);
+exec(
+    "docker-compose run -T --rm unit-tests vendor/bin/phpunit --configuration test/phpunit.xml --testsuite LorisUnitTests --coverage-html htdocs/log/codeCoverage",
```

Comment:
I don't see why this is using docker-compose to run the unit tests, it's an unnecessary complication